src/foam/nanos/auth/JWK.js

```diff
@@ -0,0 +1,42 @@
+foam.CLASS({
+    package: 'foam.nanos.auth',
+    name: 'JWK',
+    ids: [
+        'kid', 'origin'
+    ],
+    properties: [
+        {
+            class: 'String',
+            name: 'kid'
+        },
+        {
+            class: 'String',
+            name: 'use'
+        },
+        {
+            class: 'String',
+            name: 'origin'
+        },
+        {
+            class: 'Boolean',
+            name: 'trusted',
+            value: false
+        }
+    ]
+});
+
+foam.CLASS({
+    package: 'foam.nanos.auth',
+    name: 'RSA256JWK',
+    extends: 'foam.nanos.auth.JWK',
+    properties: [
+        {
+            class: 'String',
+            name: 'n'
```

Comment:
I assume that this is what these are:
n	The modulus for the [RSA public key](https://tools.ietf.org/html/rfc7518#page-30).
e	The exponent for the [RSA public key](https://tools.ietf.org/html/rfc7518#page-30).

Would be good to add as comments or documentation: